### ISSUE TITLE
Add browser hacks for mobile browsers (Safari IOS, Chrome Android)

### ISSUE THREAD

**Issue by Aarbel** [4ae9b8/browserhacks#185]
All is in the title, could you add those browsers ? 

Thanks !

**Comment by jeffclayton** [comment]
That is something we need to discuss here being that the site has not split versions on different devices. While I have been working on separating out browsers for handhelds myself (check browserstrangeness.com for my hacks for both) -- that has not been done here. It may well be time to do it.

**Comment by Aarbel** [comment]
Yes could be very cool ! (nice tips on your url, don't hesitate to make it more modern, we are talking about "appearance" webbrowser fixes :trollface: )

**Comment by jeffclayton** [comment]
If you are referring to the look and feel of my own test site, that I will have to think about-- I actually have to support extremely old browsers (in my job) so updating it and making it more modern is actually a problem - some are really outdated. At least they aren't text based...
